### DISCUSSION TITLE
About create a new uric acid detection rule

### DISCUSSION THREAD

**Discussion by jinlong418** [antismash/antismash#722] [General]
Hello,

I am currently trying to use antismash to analyze uric acid metabolism gene clusters in bacterial genomic sequences, but there are no detection rules for uric acid metabolism in the detection rule file "antismash/detection/gut_hmm_detection/cluster_rules/strict.txt". I am wondering how to design a new detection rule for this analysis and look forward to your help.
![image](https://github.com/antismash/antismash/assets/173862675/a6bef575-b1f8-475f-8c8c-1a6437e356b2)

Thank you
Jinlong

**Comment by SJShaw** [comment]
Questions about this process have come up in other contexts recently, so I've just finished off a page on the repository wiki that should hopefully help guide you in creating custom rules: https://github.com/antismash/antismash/wiki/Creating-detection-rules

**Reply by jinlong418** [reply]
Thank you very much for your guidance, which has deepened my understanding of creating custom rules.
